app/scripts/lib/account-tracker.js

```diff
@@ -382,9 +394,9 @@ export default class AccountTracker {
    * @param {number} blockNumber - the block number to update to.
    * @fires 'block' The updated state, if all account updates are successful
    */
-  async _updateForBlock(blockNumber) {
-    await this._updateForBlockByNetworkClientId(null, blockNumber);
-  }
+  #updateForBlock = async (blockNumber) => {
```

Comment:
[nit] wondering why this was made into an arrow function. Now its the only one on the controller

Reply:
`=` would make it a static method on the class

Reply:
i don't believe that is true. I think the `static` keyword defines that